DDG4/src/Geant4Kernel.cpp

```diff
@@ -94,7 +94,7 @@ Geant4Kernel::Geant4Kernel(Detector& description_ref)
   declareProperty("OutputLevel",          m_outputLevel = DEBUG);
   declareProperty("NumEvents",            m_numEvent = 10);
   declareProperty("OutputLevels",         m_clientLevels);
-  declareProperty("NumberOfThreads",      m_numThreads);
+  declareProperty("NumberOfThreads",      m_numThreads = 0);
```

Comment:
Should this be 1, or does it not matter?

Reply:
I think for single threading it does not matter. For multi threading it should be set properly.

Reply:
It is set in the constructor explicitly to "0", so in principle the re-assignment in `declareProperty(...)` is redundant.

Reply:
Ok, thanks!